### DISCUSSION TITLE
Can't find "remove alive crew menu" function in config

### DISCUSSION THREAD

**Discussion by ArtiK045** [evilfactory/traitormod#12] [Q&A] [answered]
I can't find "config.disableCrewMenu" in config file, and "Alive crew menu remover" mod breaks the game(with it traitor mod doesn't choose the traitor)

**Comment by evilfactory** [comment, marked as answer]
I just added an option to hide the crew list in the dev branch, i didn't test it very well, but hopefully there shouldn't be any issues with it.

**Reply by ArtiK045** [reply]
ok thx, ill test it enough and give info about bugs ;)

**Reply by ArtiK045** [reply]
> I just added an option to hide the crew list in the dev branch, i didn't test it very well, but hopefully there shouldn't be any issues with it.

played with it about 3 days, and didnt find any bugs, only bug with commands for bots, but it works when u aim at bot and press mouse3
